src/Sylius/Bundle/AdminBundle/Resources/config/routing/customer.yml

```
@@ -19,8 +19,10 @@ sylius_admin_customer_order_index:
         _sylius:
             section: admin
             permission: true
-            template: "@SyliusAdmin/shared/crud/index.html.twig"
+            template: "@SyliusAdmin/customer/order/index.html.twig"
             grid: sylius_admin_customer_order
+            vars:
+                customer: expr:service('sylius.repository.customer').find($id)
```

```suggestion
            vars:
                hook_prefix: 'sylius_admin.customer.order'
                customer: expr:service('sylius.repository.customer').find($id)
```

Set the prefix here and the `src/Sylius/Bundle/AdminBundle/templates/customer/order/index.html.twig` proxy file can be removed.

---

done